packages/taiko-client/prover/event_handler/assignment_expired.go

```
@@ -101,6 +101,10 @@ func (h *AssignmentExpiredEventHandler) Handle(
 
 	// If there is no contester, we submit a contest to protocol.
 	go func() {
+		if meta.IsPacaya() {
+			h.proofSubmissionCh <- &proofProducer.ProofRequestBody{Meta: meta}
```

Should we throw an ERROR instead of requesting another proof when encountering an invalid proof?

---

So should we rely on this event to start our manual fix workflow, rather than depending on specific error messages?
With this approach, since every user needs to submit a transaction, it would result in unnecessary gas cost

---

Yep, if there is an invalid, we need to pause the protocol instead of letting it verify, since it is really a rare case, the unnecessary cost is ok.